ts/Core/Series/Point.ts

```diff
@@ -194,7 +194,7 @@ class Point {
      * @name Highcharts.Point#percentage
      * @type {number|undefined}
      */
-    public percentage?: number = void 0;
+    public percentage?: number|null = null;
```

Comment:
Do we need the `null` value? In general, if there is no logical distintion between `null` and `undefined`, we should stick to `undefined`.

Reply:
I think it should be `undefined` when `point.y` is null. If it doesn't complicate the code too much.